TrenDAP/Controllers/TrenDAPDBController.cs

```diff
@@ -74,11 +74,12 @@ public class XDADataSetData
             public List<int> IDs { get; set; }
             public List<int> Phases { get; set; }
             public List<int> Groups { get; set; }
-            public List<int> Types { get; set; }
+            public List<int> Chans { get; set; }
```

Comment:
```suggestion
            public List<int> ChannelIs { get; set; }
```
I would updated naming to be clear. no need to abreviate